assets/armsnippets.jsonc

```diff
@@ -1940,6 +1940,24 @@
         ],
         "description": "Public IP Prefix"
     },
+    "Resource Group": {
+        "prefix": "arm-rg",
+        "body": [
+            "{",
+            "\t\"name\": \"${1:resourceGroup}\",",
+            "\t\"type\": \"Microsoft.Resources/resourceGroups\",",
+            "\t\"apiVersion\": \"2019-10-01\",",
+            "\t\"location\": \"[parameters('location')]\",",
+            "\t\"dependsOn\": [",
+            "\t],",
+            "\t\"tags\": [",

```

Comment:
Tags should be an object vs. array. Change the square brackets to curly braces.

https://docs.microsoft.com/en-us/azure/templates/microsoft.resources/2019-10-01/resourcegroups#microsoftresourcesresourcegroups-object